docs/docs/org-management/licensing/cloud.md

```diff
@@ -0,0 +1,112 @@
+---
+id: tooljet-cloud
+title: ToolJet Cloud
+---
+
+<div className='badge badge--primary heading-badge'>Available on: Paid plans</div>
+
+## Choosing Your ToolJet Cloud Subscription
+
+This comprehensive guide aims to assist you in the process of configuring paid plans for ToolJet Cloud. If you are looking for self-hosted/on-premises plans, please navigate to **[Self Hosted](/docs/org-management/licensing/self-hosted)** . For ToolJet Cloud, we offer three different types of subscriptions tailored to your specific requirements. Let's explore how to get the most out of ToolJet's powerful features! 🚀
+
+Let's look into three types of subscriptions:
+
+1. **Free Trial**: This is a free subscription that grants access to premium features for a 14-day trial period.
+
+   - **New Users**: Select the 14-day trial option during the onboarding process.
+   - **Existing Users**: If eligible, a start trial banner will be visible on your ToolJet dashboard.
+
+2. **Business Subscription**: This is a paid monthly subscription that you can [upgrade](#a-chosen-plan-business-plan) to from within the ToolJet app.
+3. **Enterprise Subscription**: This is a paid subscription with customizable options. To obtain this subscription, you have to contact our **[sales](https://tooljet.com/slack)** team.
+
+---
+
+## Updating To Trial Subscription (For Existing Users)
+
+If you are an existing user and wish to update to trial subscriptions, follow these steps:
+
+1. Navigate to the desired Workspace.
+2. If you are not currently on a paid plan and have not yet used your free trial, you will see a **Start Trial** banner within the ToolJet dashboard. Click on the **Start free trial** button inside this banner to initiate your 14-day trial.
+
+<div style={{textAlign: 'center'}}>
+
+<img className="screenshot-full" src="/img/licensing/dashboard-free-trial.png" alt="TJ Dashboard: Start free trial" />
+
+</div>
+
+:::caution Note
+The trial subscription will be valid for 14 days. To fully enjoy ToolJet, we recommend upgrading to premium plans within this period. If you wish to upgrade from the trial to the business or enterprise edition, you can click the **Upgrade or Renew** button or contact our team via **[Slack](https://tooljet.com/slack)**. Upon expiration, access to premium features like OpenID SSO login and Audit logs will be restricted, ensuring no data loss occurs. However, don't worry! You can still upgrade to any of our premium plans and enjoy the benefits of ToolJet.
```

Comment:
Edited version:

Your trial subscription is active for 14 days. To upgrade to our Business or Enterprise edition, simply click the Upgrade or Renew button or connect with our team on [Slack](https://tooljet.com/slack). Premium features like OpenID SSO login and Audit logs will be restricted after the trial ends. However, you can still upgrade to any of our premium plans and continue to enjoy the full benefits of ToolJet.